webpack/components/extensions/HostDetails/Tabs/TracesTab.js

```diff
@@ -110,16 +87,10 @@ const TracesTab = () => {
           } = result;
           return (
             <Tr key={id} >
-              <Td select={{
-                disable: false,
-                props: {
-                  'aria-label': `check-${application}`,
-                },
-                isSelected: selectedTraces.has(id),
-                onSelect: (event, isSelected) => onRowSelect({ isSelected, traceId: id }),
-                rowIndex,
-                variant: 'checkbox',
-              }}
+              <TdSelect id = {id}
```

Comment:
I don't think this really warrants it's own component.

Reply:
agreed